modules/backend/widgets/lists/partials/_list_head_row.php

```diff
@@ -20,7 +20,7 @@
                 <?php if ($column->width): ?>
                     style="width: <?= $column->width ?>"
                 <?php endif ?>
-                class="<?= $this->sortColumn==$column->columnName?'sort-'.$this->sortDirection.' active':'sort-desc' ?> list-cell-name-<?= $column->getName() ?> list-cell-type-<?= $column->type ?> <?= $column->getAlignClass() ?> <?= $column->headCssClass ?>"
+                class="<?= $this->sortColumn == $column->columnName ? 'sort-' . $this->sortDirection . ' active' : 'sort-desc' ?> list-cell-name-<?= $column->getName() ?> list-cell-type-<?= $column->type ?> <?= $column->getAlignClass() ?> <?= $column->headCssClass ?>"
```

Comment:
Should this be `===`?

Reply:
The original functionality was just the two `==`. It wouldn't change that, it would just ensure there's correct spacing around. I'd say leave it as is unless we're sure changing it to strict won't break anything.